src/scripts/cronjobs/emailBreachAlerts.tsx

```diff
@@ -260,7 +264,30 @@ export async function poll(
                 });
 
                 const l10n = getCronjobL10n(sanitizeSubscriberRow(recipient));
-                const subject = l10n.getString("breach-alert-subject");
+                const enabledFeatureFlags = await getEnabledFeatureFlags({
+                  email: recipient.primary_email,
+                });
+                const subject = enabledFeatureFlags.includes(
+                  "BreachEmailRedesign",
+                )
+                  ? l10n.getString("email-breach-alert-all-subject")
+                  : l10n.getString("breach-alert-subject");
+
+                /**
+                 * Without an active user session, we don't know the user's country. This is
+                 * our best guess based on their locale. At the time of writing, it's only
+                 * used to determine whether to count SSN breaches (which we don't have
+                 * recommendations for outside the US).
+                 */
+                const assumedCountryCode = getSignupLocaleCountry(recipient);
+
+                const scanData = await getLatestOnerepScanResults(
+                  recipient.onerep_profile_id,
+                );
+                const allSubscriberBreaches = await getSubscriberBreaches({
+                  fxaUid: recipient.fxa_uid,
+                  countryCode: assumedCountryCode,
+                });
```

Comment:
Nit: We would only need to fetch these when `BreachEmailRedesign` is enabled as we don’t use the data for the current email template. That shouldn’t matter too much if we decide to flip the flag soon and keep it on, though.

Reply:
That's a good idea; if these are gated behind the flag too, then we can easily disable it again [if they cause issues](https://github.com/mozilla/blurts-server/pull/5012#discussion_r1741837930).

Changed in d117dcefdda13eb94d18744146c893014636fcfa.

Reply:
Great, thanks!